serverless.ts

```diff
@@ -479,10 +495,19 @@ const serverlessConfig: AWS = {
       },
       DataBackupBucket: {
         Type: 'AWS::S3::Bucket',
-        Condition: 'IsDev',
+        Condition: 'IsDevOrProd',
         DeletionPolicy: 'Retain',
         Properties: {
           BucketName: '${self:service}-${self:provider.stage}-data-backup',
+          AccessControl: 'Private',
```

Comment:
its private by default

Reply:
I thought it would be good to make this explicit. 🤔 I can remove it